File: javaFiles/ClassAndObjects.java
package javaFiles;

public class ClassAndObjects {
    String name;

    public ClassAndObjects(String name) {
        this.name = name;
    }
}

class Main {
    public static void main(String[] args) {
            ClassAndObjects class1 = new ClassAndObjects("Visual Programming");
            ClassAndObjects class2 = new ClassAndObjects("Web Development");
            System.out.println("Class name: " + class1.name);
            System.out.println("Class name: " + class2.name);
    }
}

File: dartFiles/ClassAndObjects.dart
class Subject {
  String? name;

  Subject(String name){
    this.name = name;
  }
  // void display() {
  //   print("Name: $name.");
  // }
}

class SubjectClassTest {
  void main() {
    Subject subject = Subject("Visual Programming");
    Subject subject2 = Subject("Web Development");
    print("Name: ${subject.name}.");
    print("Name: ${subject2.name}.");
  }
}